tern/classes/package.py

```diff
@@ -40,12 +40,12 @@ def version(self, version):
         self.__version = version
 
     @property
-    def license(self):
-        return self.__license
+    def pkg_license(self):
+        return self.__pkg_license
 
-    @license.setter
-    def license(self, license):  # pylint: disable=redefined-builtin
-        self.__license = license
+    @pkg_license.setter
+    def pkg_license(self, pkg_license):  # pylint: disable=redefined-builtin
+        self.__pkg_license =pkg_license
```

Comment:
You'll need to remove the `# pylint: disable=redefined-builtin` string as well

Reply:
Done!